packages/webdriverio/src/commands/element/click.ts

```diff
@@ -123,17 +123,30 @@ export async function click(
 
     if (this.isW3C) {
         const browser = getBrowserObject(this)
-        await browser.action('pointer', {
-            parameters: { pointerType: 'mouse' }
-        })
-            .move({
-                origin: this,
-                x: xoffset,
-                y: yoffset
+        const clickNested = async () => {
+            await browser.action('pointer', {
+                parameters: { pointerType: 'mouse' }
             })
-            .down({ button })
-            .up({ button })
-            .perform(skipRelease)
+                .move({
+                    origin: this,
+                    x: xoffset,
```

Comment:
```suggestion
                    x: xoffset || 0,
```

Reply:
Let's also rename xoffset to xOffset to keep it aligned with the moveTo command

Reply:
Actually, we can use the same implementation for setting the integers in the moveTo command. Again making the two commands more alike, in which case we do not need to cast it to 0 anymore.